### DISCUSSION TITLE
アプリがmacOSで起動しません

### DISCUSSION THREAD

**Discussion by kitayama-arch** [kishikawakatsumi/TextChatTranslator#3] [Q&A]
アプリが起動しない問題に直面しております。

- **環境**
  - macOS Sonoma 14.5
  - 最新バージョンをインストール済み

- **現象**
  - Finderからアプリを起動しても何も表示されません。
  - ターミナルから `open /Applications/Text\ Chat\ Translator.app` 実行しても同様に反応がありません。
  - エラーメッセージやポップアップは表示されません。

- **対策済み事項**
  - READMEに従いアクセシビリティの許可を与えました。
  - システムの再起動を行いましたが、問題は解決しませんでした。

お忙しいところ恐れ入りますが、何か追加の対処方法やデバッグ方法をご教示いただけないでしょうか。

どうぞよろしくお願いいたします。

**Comment by stakahashi** [comment]
macOS 15 or laterですので、Sonomaでは実行できないのではないでしょうか？
https://github.com/kishikawakatsumi/TextChatTranslator?tab=readme-ov-file#requirements

**Comment by Elonreevemusk090** [comment]
@stakahashi
